ui/src/literature/components/AssignLiteratureItemDrawer.tsx

```diff
@@ -24,7 +24,7 @@ interface IAuthorResult {
 }
 
 interface AssignLiteratureItemDrawerProps {
-  visible: boolean;
+  visible: string;
```

Comment:
shouldn't be `bool`?

Reply:
I had to change this logic a little bit now that the drawer can also be displayed on literature search page and not just on details page - if this value would stay just a true/false then when ```setAssignDetailViewDrawerVisibility``` action is triggered it would render a drawer for every literature item on search page, not just the one that was clicked. So to render just one drawer component for the desired literature item I set this visible parameter to record id, and then on drawer component I check if visible parameter in state equals the selected literature item id

Reply:
if it's an id, can we use integer?

Reply:
You're right, changed to type ```number``` ✅